profiles/mercury.raml

```diff
@@ -177,6 +178,22 @@ validations:
     propertyConstraints:
       apiContract.path:
         pattern: ^((\/[a-z][a-z0-9-]*[a-z0-9]+)|(\/{[^}]+}))+$
+  
+  unique-display-name:
+    message: displayName must be unique across an API
+    targetClass: apiContract.Operation
+    functionConstraint:
+      code: |
+        function(ops) {
+          const name = (ops['core:name'] || [])[0];
+          // the validation engine offers a global accumulators hash persistent between validation function invocations
+          if (!accumulators[name]) {
```

Comment:
accumulators are new and interesting! 

should we add some rule specific namespace like "rulename:<<name>>" to avoid any conflicts?